scripts/normalize-jest-coverage.js

```diff
@@ -0,0 +1,29 @@
+const libCoverage = require('istanbul-lib-coverage');
+const { createReporter } = require('istanbul-api');
+const fs = require('fs')
+const path = require('path')
+
+const normalizeJestCoverage = (obj) => {
+  const result = { ...obj };
+
+  Object.entries(result).forEach(([k, v]) => {
+    if (v.data) result[k] = v.data;
+  });
+
+  return result;
+};
+
+const map = libCoverage.createCoverageMap();
+const jestCoverage = require('../.nyc_output/coverage-final.json');
+
+map.merge(normalizeJestCoverage(jestCoverage));
+
+const reporter = createReporter();
+reporter.addAll(['json']);
+reporter.write(map);
+
+fs.copyFile(
+  './coverage/coverage-final.json',
+  './.nyc_output/coverage-final.json',
+  (error) => { if (error) console.log(error) }
```

Comment:
This looks super readable (not suggesting a change) but could also do

```js
error => error && console.log(error) 
```

Also, by the way this is formatted I can tell your editor is missing a [Prettier integration] :) which we use for all of our JS repos based on configuration [located here](https://github.com/artsy/positron/blob/master/package.json#L206-L211)